Docs changes to reflect new assessment settings API and naming (#119)
hashicorp/terraform-docs-common: website/docs/cloud-docs/users-teams-organizations/organizations.mdx

Refer to the [variables overview](/cloud-docs/workspaces/variables) documentation for details about variable types, scope, and precedence. Refer to [managing variables](/cloud-docs/workspaces/variables/managing-variables) for details about how to create and manage variable sets.

> #### Assessments
> 
> -> **Note:** Health Assessments are a paid feature, available as part of the **Business** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
> 
> Enforce health assessments or delegate to workspace preference. Enforcing health assessments overrides workspace settings but does not change them. Refer to [workspace health assessments](/cloud-docs/workspaces/settings/health-assessments) for more information.

Reviewer: ```suggestion
<!-- BEGIN: TFC:only name:health-assessments -->
#### Assessments

-> **Note:** Health Assessments are a paid feature, available as part of the **Business** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.

Enforce health assessments or delegate to workspace preference. Enforcing health assessments overrides workspace settings but does not change them. Refer to [workspace health assessments](/cloud-docs/workspaces/settings/health-assessments) for more information.
<!-- END: TFC:only name:health-assessments -->
```

Refer to the [variables overview](/cloud-docs/workspaces/variables) documentation for details about variable types, scope, and precedence. Refer to [managing variables](/cloud-docs/workspaces/variables/managing-variables) for details about how to create and manage variable sets.

<!-- BEGIN: TFC:only name:health-assessments -->
#### Assessments

> -> **Note:** Health Assessments are a paid feature, available as part of the **Business** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.

Reviewer: ```suggestion
-> **Note:** Health Assessments are available in the [Terraform Cloud **Business** tier](https://www.hashicorp.com/products/terraform/pricing). 
```

> Enforce health assessments or delegate to workspace preference. Enforcing health assessments overrides workspace settings but does not change them. Refer to [workspace health assessments](/cloud-docs/workspaces/settings/health-assessments) for more information.

Reviewer: ```suggestion
You can enforce health assessments for all eligible workspaces or let each workspace opt in to health assessments through workspace settings. Enforcing health assessments overrides workspace settings but does not change them. Refer to [Health Assessments](/cloud-docs/workspaces/settings/health-assessments) in the workspaces documentation for more details.
```
